zuul.d/project-templates.yaml

```diff
@@ -457,6 +457,7 @@
         - build-ansible-collection:
             required-projects:
               - name: github.com/ansible-collections/kubernetes.core
+                override-checkout: 2.1.1
```

Comment:
This looks wrong to me, because we won't test any future 2.1.x versions. Can we instead branch 2.1.x and use this branch?

Reply:
Overall, what prevents us from using master here? I think this PR is connected with yours https://github.com/openshift/community.okd/pull/109, would this be a better fix?

Reply:
The problem with using openshift/community.okd#109 to fix this is that we are then forced to always keep OKD working with the current release of k8s.core. While we try to do this, there is often a lag. If we do a major release of kubernetes.core then our okd CI would be broken until we can fix it. Ideally, the voting CI jobs would use galaxy.yml for installing dependencies, but since it doesn't, I'd rather we pin the kubernetes.core version here.

Reply:
I just pushed a fix that address a breaking change that should have been caught by the CI https://github.com/ansible-collections/kubernetes.core/pull/239
This is the kind of regression that we can identify before we introduce them if we do cross-project testing. And we cannot do cross-project testing properly if we rely on the galaxy.yml, because it will fetch the last stable version.